### PR TITLE
Add meta referrer tag

### PR DESCRIPTION
**Pull request by EdOverflow**
Solves the following issue: https://hackerone.com/reports/176390

### CODE REVIEW

**Comment by whit537** [comment]
https://blog.mozilla.org/security/2015/01/21/meta-referrer/

**Comment by whit537** [comment]
lgtm

File: templates/base.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     {% block head_early %}{% endblock %}
     <title>{% if title %}{{ title }} - {% endif %}{% if banner %}{{ banner }} - {% endif %}Gratipay</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="origin" />
```
Comment (whit537) [inline templates/base.html]: Somewhat pedantic, but in the ` />` is XHTML, with HTML 5 we can end this `>`.

**Comment by whit537** [comment]
Rebased and tidied up in b76a5bbbb89baea1920ccad8a26797572bbeaa0c, will merge when green ...

File: templates/base.html (hunk 1 above)
Comment (whit537) [inline templates/base.html]: Tweaked in 7c74575cd02edb69f91f0dc8605f2331ba342500.

**Comment by EdOverflow** [comment]
Actually I would prefer we use:

~~~diff
- <meta name="referrer" content="origin" />
+ <meta name="referrer" content="no-referrer-when-downgrade" />
~~~

**Comment by whit537** [comment]
New PR! :-)

**Comment by EdOverflow** [comment]
> New PR! :-)

Tada! https://github.com/gratipay/gratipay.com/pull/4357

**Pull request merged**